frontend/src/employee-frontend/components/messages/MessageEditor.tsx

```diff
@@ -17,6 +17,7 @@ import { useTranslation } from '../../state/i18n'
 import { Bulletin } from './types'
 import { SelectorChange } from 'employee-frontend/components/messages/receiver-selection-utils'
 import MultiSelect from 'lib-components/atoms/form/MultiSelect'
+import Select from '../common/Select'
 
 type Option = {
```

Comment:
We might want to refactor this somehow. I don't really like this `Option` guy here. (This is just nit though and it's my code anyway.)